sw/host/opentitanlib/src/transport/cw310/spi.rs

```diff
@@ -89,8 +89,11 @@ impl Target for CW310Spi {
         Ok(42)
     }
 
-    fn max_chunk_size(&self) -> Result<usize> {
-        Ok(65536)
+    fn get_max_transfer_sizes(&self) -> Result<MaxSizes> {
+        Ok(MaxSizes {
+            read: 65536,
+            write: 65536,
```

Comment:
I'm not sure about this for cw310.  I'm skeptical of a large write transfer size because the PAGE_PROGRAM opcode can't accept more than 256 bytes of data at a time.

Reply:
These numbers are meant to advertise what the transport is capable of.  Clients such as `flash.rs` are responsible for knowing the capabilities of whatever chip is on the other end, and e.g. not issue dual-line opcodes if the chip does not support it, or reads/writes longer than what the chip supports.  (More logic is required in `flash.rs` to recognize e.g. erase page sizes and other capabilities, to optimize its operation.)